src/BizHawk.Client.EmuHawk/config/SoundConfig.Designer.cs

```diff
@@ -28,330 +28,409 @@ protected override void Dispose(bool disposing)
 		/// </summary>
 		private void InitializeComponent()
 		{
-			this.Cancel = new System.Windows.Forms.Button();

```

Comment:
Sorry you had to deal with the Designer. If this feature is accepted, I'll rewrite this in a declarative style, then you can rebase and have a smaller diff.